### DISCUSSION TITLE
Admin Claim: Item Frame broken by #physics? (2 related questions)

### DISCUSSION THREAD

**Discussion by FaithInMotion** [GriefPrevention/GriefPrevention#2383] [Q&A]
Pre-note: I saw the other discussions that say item frame issues may be caused by a plugin conflict and have been unreplacatable to date. I'm not sure if this is the same type of issue and my questions are not related to "What's broke", but rather "how to troubleshoot".

I am trying to troubleshoot how all item frames that had maps in them were broken in an admin-claim area. CO says it was done by #physics.

My first question is this: What constitutes "#physics"? Is there a dictionary of terms like this anywhere I could cross reference? I am trying to figure out the possible ways this could have been done.

My second question is this - is there a way with CO to see changes in an area instead of block-by-block? I have checked all single blocks in an immediate perimiter around one of the item frames and cannot find any log of any interactions/edits etc, so I cannot find the source of whatever #physics might be.

**Comment by Jikoo** [comment]
These sound like questions for CO, not GP. I haven't used CO since like 2017 when I discovered Prism, so I can't say for sure. Generally "physics" is "unknown/natural causes" aka plugin removal/walls being removed behind them. In Prism you can specify a lookup radius as part of the parameters, and I think CO has something similar, but again, CO docs/support are your friend here.

The only item frame issues are ones where people claim item frames can be damaged but then never respond when asked to isolate conflicting plugins or try with only GP installed. Presumably they have some plugin installed that un-cancels protections or doesn't fire appropriate events, but since none of them have ever replied back, no one can say for sure.
